### PR TITLE
Update neural_style_tutorial.py

### PR DESCRIPTION
**Pull request by Jiangliuer**
“normalization_mean“,”cnn_normalization_std" warning in Normalization torch.tensor(mean)

### CODE REVIEW

**Comment by holly1238** [comment]
Closing this PR. The current tutorial runs without errors.
